benchmarks/benchmark_runner.py

```diff
@@ -286,7 +286,6 @@ async def run_benchmarks(retries, benchmarks):
             results.extend(await evaluate_sample(path))
 
     summary = BenchmarkResultSummary(results)
-    os.chdir("../..")
```

Comment:
This should have been removed when sample evaluation was added. cwd is set via argument not inferred from actual cwd. This originally changed the cwd back but now there's no need.